### DISCUSSION TITLE
Daisy changing Daly bms s

### DISCUSSION THREAD

**Discussion by Maxmat99** [mr-manuel/venus-os_dbus-serialbattery#85] [Q&A]
Hi! I have just had to re flash my sd card on pi , and saw that there were mentions of daisy chaining ! That is cool! What will this involve ?
How is development going? 
6x Daly bms  
4 x multi plus
Pi 
Off grid system with pv inverters and mppt
Keep up the good work!

**Comment by mr-manuel** [comment]
You will need only one cable to connect multiple BMS, but the data will update slower, since they use the same connection.
Development is only waiting for people to confirm that it works.

**Reply by Maxmat99** [reply]
Sounds great, if you need help testing I am willing to help… just tell me what I need!!

**Reply by mr-manuel** [reply]
https://mr-manuel.github.io/venus-os_dbus-serialbattery_docs/next/general/install#daly-bms

**Reply by Maxmat99** [reply]
Ok that’s cool, are the bms cables just spliced together on the same adapter then?

**Reply by mr-manuel** [reply]
You have to follow the RS485/Modbus rules. Best when you Google for it.

**Reply by Maxmat99** [reply]
Yup ok that’s great I’ll give it a go! Nice one! Cuts down on usb hub connections everywhere , I am also assuming that isolating the ground is necessary , and only using the a+ b-  ….
